SPHINXsys/src/shared/bodies/body_relation.h

```diff
@@ -171,8 +174,8 @@ namespace SPH
 	class BaseBodyRelationContact : public SPHBodyRelation
 	{
 	protected:
-		StdVec<MeshCellLinkedList*> target_mesh_cell_linked_lists_;
-		StdVec<SearchRangeMultiResolution*> get_search_ranges_;
+		StdVec<CellLinkedList*> target_cell_linked_lists_;
```

Comment:
For example unique pointers here, and these kind of places

Reply:
For geometric modeling using smart pointer is a good idea.